_episodes/03-registry.md

```diff
@@ -9,35 +9,62 @@ objectives:
 keypoints:
 - "Registring your code in a community registry makes your code findable for others."
 ---
-The [third principle](https://fair-software.eu/recommendations/registry) states:
-> ## Principle 3
+
+> ## [Recommendation 3](https://fair-software.eu/recommendations/registry)
+>
 > Register your code in a community registry
 {: .callout}
+![recommendation3]({{ page.root }}/fig/recommendation3.png)
+
+## Why registering your Software is important
+
+Registering your software makes it easier for others to find it,
+particularly through the use of search engines such as Google.
+Community registries typically employ metadata to describe each software package.
```

Comment:
Not everyone knows what metadata is. Maybe you can add a sentence on it. I like the description from Lifewire 'Metadata is data about data. In other words, it's information that's used to describe the data that's contained in something like a web page, document, or file.' after https://www.lifewire.com/metadata-definition-and-examples-1019177

Reply:
Thank you. I added this definition to the next callout.